tests/util/model.py

```diff
@@ -49,7 +49,7 @@ def sample(self, query_points: TensorType, num_samples: int) -> TensorType:
         mean, var = self.predict(query_points)
```

Comment:
So if query_points is of always shape [..., N, D] should this be asserted somewhere, to catch someone passing in something with rank < 2? (comment not specific to this ProbabilisticModel)